core/src/test/java/io/parsingdata/metal/ShorthandsTest.java

```diff
@@ -214,4 +218,28 @@ public void checkSeqTokens() {
         assertEquals(DEFB, seq.tokens.tail.head);
     }
 
+    final ParseGraph PARSEGRAPH = ParseGraph.EMPTY.add(createParseValue("a", 126)).add(createParseValue("a", 84)).add(createParseValue("a", 42));
+
+    @Test
+    public void mapLeftWithSub() {
```

Comment:
Possibly, but we decided to not go for carthesian product in general `BinaryValueExpression` handling because the result would be too difficult to navigate declaratively. I think that will be the case here as well.

But I do agree that expanding a multiple value list breaks the concept of map (because it doesn't map everything in tjat situation...)